checkov/terraform/graph_builder/graph_components/module.py

```diff
@@ -151,7 +151,10 @@ def _add_resource(self, blocks: List[Dict[str, Dict[str, Any]]], path: str) -> N
                     attributes = self.clean_bad_characters(resource_conf)
                     if not isinstance(attributes, dict):
                         continue
-                    has_dynamic_block = handle_dynamic_values(attributes)
+                    if os.getenv('RENDER_DYNAMIC_MODULES', 'True').lower() == 'false':
```

Comment:
```suggestion
                    if os.getenv('CHECKOV_RENDER_DYNAMIC_MODULES', 'True').lower() == 'false':
```
not that some other tool, also uses such an env var 😄 and can you move it to either global level or class attribute, so we don't evaluate it for each resource